packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -10,18 +10,38 @@ export interface DrawerPanelContentProps extends React.HTMLProps<HTMLDivElement>
   children?: React.ReactNode;
   /* Flag indicating that the drawer panel should have a border. */
   hasBorder?: boolean;
+  /* Default width for drawer panel */
+  width?: 25 | 33 | 50 | 66 | 75 | 100;
+  /* Drawer panel width on large viewports */
+  widthOnLg?: 25 | 33 | 50 | 66 | 75 | 100;
+  /* Drawer panel width on xl viewports */
+  widthOnXl?: 25 | 33 | 50 | 66 | 75 | 100;
+  /* Drawer panel width on 2xl viewports */
+  widthOn2Xl?: 25 | 33 | 50 | 66 | 75 | 100;
 }
 
 export const DrawerPanelContent: React.SFC<DrawerPanelContentProps> = ({
   className = '',
   children,
   hasBorder = false,
+  width,
+  widthOnLg,
+  widthOnXl,
+  widthOn2Xl,
   ...props
 }: DrawerPanelContentProps) => (
   <DrawerContext.Consumer>
     {({ isExpanded }) => (
       <div
-        className={css(styles.drawerPanel, hasBorder && styles.modifiers.border, className)}
+        className={css(
+          styles.drawerPanel,
+          hasBorder && styles.modifiers.border,
```

Comment:
I looked at core and it also has a `pf-m-no-border` modifier, are we missing that? maybe out of context for this PR?

Reply:
@jschuler good catch - that wasn't part of this issue, but we can open a new issue to get that added & update the demo & test

Reply:
ok thanks, opened issue to track this https://github.com/patternfly/patternfly-react/issues/3983